server/fishtest/util.py

```diff
@@ -329,10 +330,33 @@ def estimate_game_duration(tc):
 def remaining_hours(run):
     r = run["results"]
     if "sprt" in run["args"]:
-        # current average number of games. Regularly update / have server guess?
-        expected_games = 58000
-        # checking randomly, half the expected games needs still to be done
-        remaining_games = expected_games / 2
+        # Current average number of games. The number should be regularly updated.
+        average_total_games = 95000
+
+        # SPRT tests always have pentanomial stats.
+        assert "pentanomial" in r.keys()
+        p = r["pentanomial"]
+        N = sum(p)
+
+        sprt = run["args"]["sprt"]
+        alpha, beta = sprt["alpha"], sprt["beta"]
+        llr = sprt["llr"]
+        o0, o1 = 0, 0
+        if "overshoot" in sprt:
+            o = sprt["overshoot"]
+            o0 = -o["sq0"] / o["m0"] / 2 if o["m0"] != 0 else 0
+            o1 = o["sq1"] / o["m1"] / 2 if o["m1"] != 0 else 0
+        if abs(llr) < 1e-4:
+            llr = math.copysign(1e-4, llr)
+        llr_bound = (math.log(beta / (1 - alpha)) - o1 if llr > 0 else
+                     math.log((1 - beta) / alpha) + o0)
+
+        # Assume all tests use default book (UHO_XXL_+0.90_+1.19).
+        book_positions = 223070
+        t = scipy.stats.beta(1, 15).cdf(min(N / book_positions, 1.0))
+        expected_games_llr = int(2 * N * llr_bound / llr)
```

Comment:
IMO this should be:
```python
expected_games_llr = int(abs(2 * N * llr_bound / llr))
```

Reply:
I forgot to fix the order between two bounds after changing the sign.
However, I'll add the abs in case something going wrong.